BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTests.cs

```diff
@@ -24,4 +31,37 @@ public void Index_WhenCalledByAnonymousUser_ReturnsViewWithLoginLink()
         result!.ViewData.Model.Should().BeOfType<AdminViewModel>()
             .Which.CurrentUser.Should().BeNull();
     }
+
+    [Fact]
+    public void GetAdminList_ReturnsViewWith_AdminListModel()
+    {
+        // Arrange
+        var fakeAdminService = A.Fake<IAdminService>();
+        var fakeListOfAdmins = A.Dummy<Task<(List<UserDbModel> CurrentAdmins, List<UserDbModel> PendingAdmins)>>();
+        var fakeAdminController = new AdminController(null!, null!, fakeAdminService);
+        A.CallTo(() => fakeAdminService.GetAdminLists()).Returns(fakeListOfAdmins);
+
+        // Act
+        var result = fakeAdminController.GetAdminList().Result as ViewResult;
+        var resultService = fakeAdminService.GetAdminLists();
+
+        // Assert
+        result!.Model.Should().BeOfType<AdminListModel>();
+        resultService.Should().Be(fakeListOfAdmins);
+    }
+
+    [Fact]
+    public void AcceptAdminRequest_Calls_ApproveAdminAccessRequest()

```

Comment:
I would rename this test something like `AcceptAdminRequest_WhenCalledWithAdminPermissions_ApproveAdminAccessRequest()`.

We should have tests to check this endpoint executes properly when called by an admin, and that it fails/returns an error/etc. when called by someone without the relevant permissions.